tests/scripts/task_convert_scripts_to_python.sh

```diff
@@ -17,6 +17,9 @@
 # under the License.
 set -euxo pipefail
 
+# SKIP: DO NOT UPSTREAM TO MAIN
+exit 0
```

Comment:
Can you move this logic into the Jenkins scripts themselves before any `node(...)` calls are made? Doing it here still allocates machines, runs the docker setup, etc. which is not that much but expensive to end up doing nothing